OCPBUGS34817: Infrastructure node workloads 'taints' are ambiguous in example and description (#90893)
openshift/openshift-docs: modules/creating-an-infra-node.adoc

See Creating infrastructure machine sets for installer-provisioned infrastructure environments or for any cluster where the control plane nodes are managed by the machine API.
====

> Requirements of the cluster dictate that infrastructure, also called `infra` nodes, be provisioned. The installation program provisions only control plane and worker nodes. Worker nodes can be designated as infrastructure nodes through labeling. You can then use taints and tolerations to move appropriate workloads to the infra nodes. See "Moving resources to infrastructure machine sets".

Reviewer: ```suggestion
Requirements of the cluster dictate that infrastructure (infra) nodes, be provisioned. The installation program provisions only control plane and worker nodes. Worker nodes can be designated as infra nodes through labeling. You can then use taints and tolerations to move appropriate workloads to the infra nodes. For more information, see "Moving resources to infrastructure machine sets".
```
In the docs, it look like most instances of "infra node" do not place the term "infra" in backticks. Within this PR, I see the phrase both with "infra" in backticks and without. I'm not sure if one formatting style is more accurate than the other, but it would be good to be consistent.

> You can optionally create a default cluster-wide node selector. The default node selector is applied to pods created in all namespaces. This creates an intersection with any existing node selectors on a pod, which additionally constrains the pod's selector.

Reviewer: I'm not entirely sure what "This" refers to in the third sentence.

> [IMPORTANT]

Author: **PEER REVIEWER** I copy/pasted this admonition from [below](https://github.com/openshift/openshift-docs/pull/90893/files#diff-e4fc54e628c59a7e372757c78f9663c9c6284eae1b1c628c335345628e95db75L43-L50). No new text in this note.

.. Save the file to apply the changes.

> You can now move infrastructure resources to the newly labeled `infra` nodes. Also, remove any workloads that you do not want, or do not belong, on the new infrastructure node. See the list of workloads supported for use on infrastructure nodes in "{product-title} infrastructure components".

Reviewer: See previous comment about whether "infra" should be in backticks.

> You can now move infrastructure resources to the newly labeled `infra` nodes. Also, remove any workloads that you do not want, or do not belong, on the new infrastructure node. See the list of workloads supported for use on infrastructure nodes in "{product-title} infrastructure components".

Reviewer: ```suggestion
You can now move infrastructure resources to the newly labeled infra nodes. Also, remove any workloads that you do not want, or that do not belong, on the new infra node. See the list of workloads supported for use on infrastructure nodes in "{product-title} infrastructure components".
```
See earlier comment about whether to put "infra" in backticks.
